canonical-web-and-design/ubuntu.com
# 1224 japanese uasd update

## Done
Update Japanese UASD page

## QA
- Check out this feature branch
- Run the site using the command ./run serve
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-description/ja](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/ja)
- Run through the following QA steps
- Compare to [brief](https://docs.google.com/document/d/1U3MfSVmaQSqSK9MXfpy--N65oBOsoWwUpQfGJ5ZCaPI/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1224
